metrics/firebase/test/firestore/rules/test_utils/test-data.js

```diff
@@ -34,15 +45,21 @@ const builds = {
     workflowName: "workflow",
     duration: 345,
     url: "url2",
-    coverage: 1.0
+    coverage: 1.0,
   },
 };
 
+exports.projectGroups = projectGroups;
 exports.builds = builds;
 
 /** A firebase user needed for tests */
 exports.user = { uid: "uid" };
 
+/** Get a test project group */
+exports.getProjectGroup = function () {
+  return JSON.parse(JSON.stringify(projectGroups["project_groups/1"]));
```

Comment:
What if we create reusable methods for a deep and shallow copy to highlight the meaning of JSON.parse(... and Object.assign?